packages/wonder-blocks-form/src/components/__tests__/labeled-text-field.test.js

```diff
@@ -186,6 +187,49 @@ describe("LabeledTextField", () => {
         expect(input).toBeDisabled();
     });
 
+    it("ariaDescribedby prop sets aria-describedby", () => {
+        // Arrange
+        const ariaDescription = "aria description";
+
+        // Act
+        render(
+            <LabeledTextField
+                label="Label"
+                value=""
+                onChange={() => {}}
+                ariaDescribedby={ariaDescription}
+            />,
+        );
+
+        // Assert
+        const input = screen.getByRole("textbox");
+        expect(input.getAttribute("aria-describedby")).toEqual(ariaDescription);
```

Comment:
Nit: You can also use a custom matcher `expect(input).toHaveAttribute("aria-describedby", ariaDescription)`. No need to change it, but just wanted to share in case you find it helpful.